examples/04_manipulating_images/plot_negate_image.py

```diff
@@ -0,0 +1,34 @@
+"""
+Negate image
+============
+
+Here we compute a negative image by multiplying it's voxel values with -1.
```

Comment:
Also say that the purpose of this example is to illustrate math_img.

It's important to always guide the reader so that he or she knows why he is reading an bit of code.